ui/src/widgets/ui-slider/UISlider.vue

```diff
@@ -68,6 +81,17 @@ export default {
     },
     mounted () {
         this.value = this.messages[this.id]?.payload
+        // Set default colors if none are chosen
+        const themePrimaryColor = 'rgb(' + getComputedStyle(document.documentElement).getPropertyValue('--v-theme-primary') + ')'
+        if (this.color.trim() === '') {
+            this.dynamic.color = themePrimaryColor
+        }
+        if (this.colorTrack.trim() === '') {
+            this.dynamic.colorTrack = themePrimaryColor
+        }
+        if (this.colorThumb.trim() === '') {
+            this.dynamic.colorThumb = themePrimaryColor
+        }
```

Comment:
Because of [how we override the Vuetify theming](https://github.com/FlowFuse/node-red-dashboard/blob/0741c0d76f9df60240dffa1e25d3218a29921187/ui/src/layouts/Baseline.vue#L205) with the NR-defined colour set, this isn't a reliable way of getting the actual value set here...

We also actually don't need to have the dynamic setting here either, as it falls back to `null`, which then uses our CSS fix (left in another review comment) do it's magic.

```suggestion
```